README.md

```diff
@@ -119,7 +119,7 @@ Follow the following steps to get started.
     ```bash
     aws route53 list-hosted-zones-by-name --dns-name <YOUR_DOMAIN_NAME> --query 'HostedZones[0].Id' --output text | cut -d'/' -f3
     # in the setups/config file, update the zone id.
-    HOSTEDZONE_ID=ZO020111111
+    HOSTEDZONE_ID: ZO020111111
```

Comment:
this should actually be `hosted_zone_id: ` if we want to get this right. can you update the PR plz?

Reply:
@nimakaviani Yes my bad, fixed it, Please merge this. Thanks!